index.tsx

```diff
@@ -16,3 +16,4 @@ export { compile, exports, hast, run, mdast, mdastV6, mdx, migrate, plain, remar
 export { default as Owlmoji } from './lib/owlmoji';
 export { Components, utils };
 export { tailwindCompiler } from './utils/tailwind-compiler';
+export { regex as gemojiRegex } from './processor/transform/gemoji+';
```

Comment:
what's this plus do?

Reply:
Nothing. I think I was being clever, like it's github emojis plus some.

Reply:
ah ok, i thought it was some webpack thing